src/main/kotlin/lotto/view/ManualLottoView.kt

```diff
@@ -0,0 +1,40 @@
+package lotto.view
+
+import lotto.domain.LottoNumbers
+import lotto.domain.LottoTicket.ManualLottoTicket
+import lotto.domain.LottoTickets
+import lotto.view.util.splitInputNumbersCommand
+
+object ManualLottoView {
+    fun inputManualLottoCount(maxPurchaseLottoCount: Int): Int {
+        println("\n수동으로 구매할 로또 수를 입력해 주세요.")
+        val inputManualLottoCountCommand: String? = readlnOrNull()
+        requireNotNull(inputManualLottoCountCommand) { "입력된 수동 로또 개수가 없습니다." }
+
+        val inputManualLottoCount = inputManualLottoCountCommand.toIntOrNull()
+        requireNotNull(inputManualLottoCount) { "구입금액이 올바르게 입력되지 않았습니다." }
+        require(inputManualLottoCount >= 0) { "수동 로또 개수는 0개부터 입력 가능합니다." }
+        require(inputManualLottoCount <= maxPurchaseLottoCount) { "구입 가능한 수동 로또 개수를 초과했습니다." }
+
+        return inputManualLottoCount
+    }
+
+    fun repeatInputManualLottoNumbers(manualLottoCount: Int): LottoTickets {
+        println("\n수동으로 구매할 번호를 입력해 주세요.")
+
+        val lottoNumbersList = mutableListOf<LottoNumbers>()
+        repeat(manualLottoCount) { lottoNumbersList.add(inputManualLottoNumbers()) }
```

Comment:
이런식으로도 작성이 가능해보이네요! :) 
```suggestion
        val lottoNumbersList = List(manualLottoCount) { inputManualLottoNumbers() }
```

Reply:
제안해 주신 대로 코드를 작성하면 한 줄로 더 간결하게 표현할 수 있겠네요!

map을 통해 ManualLottoTicket을 생성하는 부분이 바로 리스트 초기화와 결합되어 코드가 더 깔끔해진 것 같습니다.

가독성을 높일 수 있는 좋은 예시를 제안해 주셔서 감사합니다.